models/yang/sonic/sonic-neigh.yang

```diff
@@ -0,0 +1,53 @@
+module sonic-neigh {
+    namespace "http://github.com/Azure/sonic-neigh";
+    prefix sneigh;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC yang for NEIGH_TABLE";
+
+    revision 2019-10-10 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-neigh {
+
+        container NEIGH_TABLE {
+
+            list NEIGH_TABLE_LIST {
```

Comment:
Is this a read-only table ? We should then use 'config false'.

Reply:
Sure!